tests/mock_data/tasking_data/get_quotations_decision_ACCEPTED.json

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
Naive question: How did you generate these mock responses?

Reply:
Not naive at all. This is a library that I do not find it very friendly. Please see the docs here: https://readthedocs.org/projects/requests-mock/downloads/pdf/latest/